File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--updating-a-chart-in-a-document-using-radio-buttons_7.cs
        private void ChartOptions1_SelectionChanged(object sender, EventArgs e)
        {
            try
            {
                object index = 1;
                Word.Shape shape = this.Shapes.get_Item(ref index);

                // Activate the shape.
                shape.OLEFormat.Activate();

                Microsoft.Office.Interop.Graph.Chart dataChart = 
                    (Microsoft.Office.Interop.Graph.Chart)shape.OLEFormat.Object;
                dataChart.ChartType = this.ChartOptions1.Selection;

                // Deactivate the shape.
                this.ChartOptions1.Select();
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message);
            }
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--updating-a-chart-in-a-document-using-radio-buttons_7.vb
    Private Sub ChartOptions1_SelectionChanged(ByVal sender As Object, ByVal e As System.EventArgs) _
        Handles ChartOptions1.SelectionChanged

        Try
            Dim shape As Word.Shape = Me.Shapes.Item(1)

            ' Activate the shape.
            shape.OLEFormat.Activate()

            Dim dataChart As Graph.Chart = CType(shape.OLEFormat.Object, Graph.Chart)
            dataChart.ChartType = Me.ChartOptions1.Selection

            ' Deactivate the shape.
            Me.ChartOptions1.Select()

        Catch ex As Exception
            MessageBox.Show(ex.Message)
        End Try
    End Sub